test/resources/deployment.go

```diff
@@ -100,6 +100,18 @@ func WaitForDeploymentReady(kubeClient kubernetes.Interface, name, namespace str
 	return wait.PollImmediate(interval, timeout, verifyFunc)
 }
 
+func WaitForDeploymentAvailable(kubeClient kubernetes.Interface, name, namespace string, interval, timeout time.Duration) error {
```

Comment:
I think `WaitForDeploymentReady` does the same

Reply:
`WaitForDeploymentReady` waits if no of replicas are same as mentioned in spec. `WaitForDeploymentAvailable` waits if deployment status is `Available` now